expense-tracker/src/db/procedures/fetch_user_details_v1dot0.sql

```diff
@@ -1,6 +1,6 @@
 DELIMITER $$
 create procedure fetch_user_details_v1dot0 (
-    IN in_user_id bigint
+    IN in_user_id varchar(50)
```

Comment:
Rename this to in_emal_id itself. Also the size of the email id column should be consistent  across everywhere. So rename this to in_email_id varchar(100). ( I see 100 as the size of email id column)